src/app/clients/page.tsx

```diff
@@ -9,16 +9,20 @@ const dataFetch: () => Promise<Schema["clients"][] | null> = async () => {
 };
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data = await dataFetch();
+    const data: Datum[] | null = (await dataFetch()) ?? [];
```

Comment:
Our function assumes the input data to have type array so that we can perform array-related functions like indexing for instance. Null object will throw an error but empty array won't so we have to keep it